src/features/events/components/newsletter/newsletter.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { shallow, mount } from 'enzyme'
+import NewsletterForm from './'
+
+describe('<NewsletterForm/>', () => {
+  beforeAll(() => {
+    global.___loader = {
+      enqueue: jest.fn(),
+    }
+    global.fetch = jest.fn().mockImplementation(() => Promise.resolve({}))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["Promise" is not defined.](https://app.codacy.com/manual/PrideInLondon/pride-london-web/pullRequest?prid=4755583)